client/db/bolt/db.go

```diff
@@ -124,6 +124,8 @@ func NewDB(dbPath string, logger dex.Logger) (dexdb.DB, error) {
 	_, err := os.Stat(dbPath)
 	isNew := os.IsNotExist(err)
 
+	logger.Infof("opening database at: %s", dbPath)
+
```

Comment:
Nice, we could have it after opening and doing the necessary tasks. i.e right before we return `bdb` and even include the version.

```
bdb.log.Infof("Opened database (version = %d, file = %s)", dbVersion, dbPath)
```

Reply:
Ok, adjusted logged messages a bit.